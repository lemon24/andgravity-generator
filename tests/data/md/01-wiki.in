[[link]]
[[link|title]]
[[ link ]]
[[ link | title ]]
