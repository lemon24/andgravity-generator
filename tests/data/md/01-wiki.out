<p><a href="LINK">link</a>
<a href="LINK">title</a>
<a href="LINK">link</a>
<a href="LINK">title</a></p>
